### PR TITLE
M3 Afloat

### PR DESCRIPTION
**Pull request by garcia-alejandro**
# Description
Afloat M3

* Created the tree component to handle the information of the root and child
* Refactoring Tax Credit details to navigate through the tree to get the information about the Tax Credit
* The Offer table was created to handle the offers and the calls to the extrinsic and the validations
* Refactoring the details tax credits to use the instance ID

### Tickets
* https://github.com/hashed-io/afloat-web3-ui/issues/4
* https://github.com/hashed-io/afloat-web3-ui/issues/5
* https://github.com/hashed-io/proxy-financial-UI/issues/360
* https://github.com/hashed-io/afloat-web3-ui/issues/6

### CODE REVIEW

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -55,7 +55,8 @@ jobs:
           IPFS_PROJECT_SECRET: '${{ secrets.IPFS_PROJECT_SECRET }}'
           FAUCET_SERVER_URL: 'https://faucet-dev.hashed.systems'
           GATED_PALLET_ID: '0x5d208df71427902157988a56e7527474a7dce921f9d5efa159dcfc849289a53c'
-          GATED_MARKETPLACE_ID: '0x1feda8a89ef968df5fbe082e4c48f88eed4dcb0ec11ca12d033e963e72594873'
+          GATED_MARKETPLACE_ID: '0xb3914e1762ff9ca8caec57959715350885632a8a323b8c160fc814600669ad33'
```
Comment (jmgayosso) [inline .github/workflows/deploy-dev.yml:58]: Is this a remote marketplace id???

File: src/boot/api.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ import { showGlobalLoading, hideGlobalLoading, showGlobalNotification } from '~/
 // const { NbvStorageApi } = require('../../../nbv-client-api')
 import { NbvStorageApi } from '@jmgayosso/nbv-client-api'
 
-// const { AfloatApi } = require('../../../afloat-client-api')
-import { AfloatApi } from '@jmgayosso/afloat-client'
+const { AfloatApi } = require('../../../afloat-client-api')
```
Comment (jmgayosso) [inline src/boot/api.js]: Replace this static require by package

File: src/boot/api.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ export default async ({ app, store }) => {
 
     // Hashed Confidential Docs
     const ipfsAuthHeader = `Basic ${Buffer.from(`${process.env.IPFS_PROJECT_ID}:${process.env.IPFS_PROJECT_SECRET}`).toString('base64')}`
+    console.log({ wss: process.env.WSS })
```
Comment (jmgayosso) [inline src/boot/api.js]: remove this console log

File: src/components/marketplace/marketplace-item.vue [hunk 4]
```diff
@@ -7,7 +7,10 @@ q-card(
   .row.justify-center
     q-icon(name="store" size="4em" color="primary")
   #labels
-    .text-subtitle2.text-weight-bold {{ marketplace?.value.label }}
+    .row.justify-center
+      .text-subtitle2.text-weight-bold {{ marketplace?.value.label }}
+    .row.justify-center
+      .text-caption.text-weight-bold.text-red fee: {{ getFee}}
```
Comment (jmgayosso) [inline src/components/marketplace/marketplace-item.vue]: here is missing a space before close {}

File: src/pages/login/SignIn.vue [hunk 5]
```diff
@@ -0,0 +1,131 @@
+<template lang='pug'>
```
Comment (jmgayosso) [inline src/pages/login/SignIn.vue]: where is used this page???

**Review by jmgayosso — COMMENTED** [review]
I wrote a few comments, also I have a question (view comments), please let me know any comment or question

File: .github/workflows/deploy-dev.yml (hunk 1 above)
Comment (garcia-alejandro) [reply .github/workflows/deploy-dev.yml:58]: Is the ID of the M3 of Afloat. It is used to point to the marketplace that is for Afloat [Afloat Tab].

So with this ID, the UI gets the information of the Afloat Marketplace without getting others' market info

@jmgayosso

File: src/boot/api.js (hunk 2 above)
Comment (garcia-alejandro) [reply src/boot/api.js]: Sure! I'll use the new version of package 0.0.6

File: src/boot/api.js (hunk 3 above)
Comment (garcia-alejandro) [reply src/boot/api.js]: Got it. Thanks!

File: src/components/marketplace/marketplace-item.vue (hunk 4 above)
Comment (garcia-alejandro) [reply src/components/marketplace/marketplace-item.vue]: Got It. Thanks!

File: src/pages/login/SignIn.vue (hunk 5 above)
Comment (garcia-alejandro) [reply src/pages/login/SignIn.vue]: Is not used on the UI. I'll remove the component.

**Review by jmgayosso — APPROVED** [review]

**Pull request merged**